src/main/resources/static/api.yml

```diff
@@ -2855,3 +3042,230 @@ components:
         - pending
         - active
         - archived
+    AssessmentId:
+      description: The ID of assessment being used. This should always be the latest Layer 3 assessment, regardless of state.
+      type: long
+      example: 138985987
+    OASysOffence:
+      type: object
+      properties:
+        date:
+          type: string
+          format: date
+          example: "1985-06-15"
+        name:
+          type: string
+          example: "Manslaughter: Diminished responsibility"
+    ArrayOfOASysOffenceAnalysisQuestions:
```

Comment:
OpenAPI Generate doesn't seem to like these (it's outputting properties of this type as `List<Any>` for some reason.)  Defining just the object here then putting it in an array wherever it's used seems to make it generate properly.

Reply:
I think this is resolved now, thanks. The generator is now generating `List<OASysQuestion>` rather than[ List<Any>](url), e.g.

`@field:JsonProperty("offenceAnalysis", required = true) val offenceAnalysis: kotlin.collections.List<OASysQuestion>`

Did I understand correctly?

Reply:
👍 you did indeed!